admin/react/assets/src/dashboard-app/pages/welcome/GridItem.js

```diff
@@ -0,0 +1,143 @@
+import React, { useState, useEffect } from "react";
+import { __ } from "@wordpress/i18n";
+import apiFetch from '@wordpress/api-fetch';
+import EditFont from "../fonts/edit/EditFont";
+import { useSelector, useDispatch } from 'react-redux';
+
+const ListItem = ({ item }) => {
+	const [checkDelete, setCheckDelete] = useState(false);
+
+	const [editFontId, setEditFontId] = useState(item.id);
+	const [editFontType, setEditFontType] = useState(item['font-type'] ? item['font-type'] : 'local');
+	const [editFontName, setEditFontName] = useState(item.title);
+	const [openPopup, setOpenPopup] = useState(false);
+
+	const updatedEditData = useSelector( ( state ) => state.editFont );
+	const [editFontData, setEditFontData] = useState(updatedEditData);
+
+	const dispatch = useDispatch();
+
+	const setupEditFontScreen = (e) => {
+		const fontId = e.target.dataset.font_id;
+		const fontType = e.target.dataset.font_type;
+		const fontName = e.target.dataset.font_name;
+
+		setOpenPopup(! openPopup);
+		setEditFontId(fontId);
+		setEditFontType(fontType);
+		setEditFontName(fontName);
+	}
+
+	useEffect( () => {
+		if ( openPopup ) {
+			setEditFontData( editFontData );
+			dispatch( { type: 'SET_EDIT_FONT', payload: editFontData } );
+		} else {
+			setEditFontData( null );
+			dispatch( { type: 'SET_EDIT_FONT', payload: null } );
+		}
+	}, [openPopup] );
+
+	const deleteFontPost = ( e ) => {
+		console.log( '***** Deleting Font *****' );
```

Comment:
Remove console.log if not required @imnavanath

Reply:
Removed